main.py

```diff
@@ -0,0 +1,22 @@
+"""
+Week 8
+This is to test how many hours a grocery store is open a week
+"""
+
+class Grocery_Store:
+    """
+    bla
+    """
+    def __init__(self, hours, days):
+        self.hours = hours
+        self.days = days
+
+def total_hours(self):
```

Comment:
You'll want to indent the `def total_hours` (and the code under that function) so that it is a part of the `GroceryStore` class:

```
class GroceryStore:
    def __init__(self):
        pass # your logic here

    def total_hours(self):
        pass # your logic here
```

`-1pt`